.github/workflows/ci.yml

```diff
@@ -215,7 +215,7 @@ jobs:
       - name: start database, server, and app
         run: |
           echo "==> start database and data containers"
-          docker-compose -f "${{ env.DOCKER_COMPOSE_PATH }}" up -d
+          docker compose -f "${{ env.DOCKER_COMPOSE_PATH }}" up -d
```

Comment:
`docker-compose` refers to an older, standalone tool (Compose v1), whereas `docker compose` refers to the new, integrated version built directly into Docker (Compose v2), as described [here](https://docs.docker.com/compose/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2). Newer Docker Desktop versions [alias](https://docs.docker.com/compose/migrate/#what-does-this-mean-for-my-projects-that-use-compose-v1) `docker-compose` to redirect commands to `docker compose`. However, users with older Docker Desktop versions may run into issues when v1 is used instead of v2. Change scripts to use `docker compose` instead of `docker-compose` so Compose v2 will be used.

Reply:
Updated workflow still runs as expected -- https://github.com/hallieswan/Agora/actions/runs/9749882114/job/26908035251